README.md

```diff
@@ -138,26 +138,20 @@ To update Keycloak with, for example, a new client:
 
 To run, build and test locally:
 
-1. Copy the [tdr-realm-export.json](https://github.com/nationalarchives/tdr-configurations/blob/master/keycloak/tdr-realm-export.json) from the tdr-configurations repository into the tdr-auth-server directory
-2. Navigate to the cloned repository: `$ cd tdr-auth-server`
-3. Build the TDR theme:
-    * If npm is not installed install [nvm](https://github.com/nvm-sh/nvm#intro) in root directory
-    * Once nvm is installed run: `[root directory] $ nvm install 16.17.0`
-    * Run the following commands in the root directory:  `[root directory] $ npm install` and `[root directory] $ npm run build-theme`
-        * this will compile the theme sass, copy the static assets to the theme `resource` directory and compile the typescript for WebAuthn.
-4. Build all three spi jars:
-    * From the root directory run the following command: `sbt assemblyPackageDependency assembly`
-    * This will generate the jar for the GovUK Notify service and the Event Publisher service
-5. Update the start script. In the `import_tdr_realm.py` replace:
+1. Navigate to the cloned repository: `$ cd tdr-auth-server`
+2. If npm is not installed, install [nvm](https://github.com/nvm-sh/nvm#intro) in the root directory
+3. Run the `local_setup.sh` script in root directory by running the command `$ sh local_setup.sh` (you can replace "sh" with whatever shell you use)
```

Comment:
Minor nitpick but you've got the shebang set as `/bin/bash` and you're running it with `sh` here. You can just do `./local_setup.sh` and that should run with bash. It doesn't matter really it's just a bit neater.